src/app/admin/backup.ts

```diff
@@ -93,7 +104,18 @@ export const uploadBackup =
             addEventToCurrentSpan("Static channel backup to GoogleCloud successful.")
           } catch (error) {
             logger.error({ error }, "Static channel backup to GoogleCloud failed.")
-            recordExceptionInCurrentSpan({ error: error, level: ErrorLevel.Warn })
+            if (error instanceof Error) {
```

Comment:
You have duplicated this logic 3 times in this file - please push it down into `recordExceptionInCurrentSpan`